dataobj/schedule_entry.h

```diff
@@ -47,7 +47,10 @@ struct schedule_entry_t
 		UNLOAD_ALL        = 1U << 5, // The convoy unloads all loads here.
 		LOAD_BEFORE_DEP   = 1U << 6, // The convoy loads just before the departure.
 		TRANSFER_INTERVAL = 1U << 7,
-	};
+		REVERSE_CONVOY	= 1U << 8,// The reverse order of vehicles in convoy
```

Comment:
[nit] The format and comment
```suggestion
		REVERSE_CONVOY	= 1U << 8, // The convoy reverses the order of its vehicles.
```